sql/catalyst/src/main/java/org/apache/spark/sql/connector/expressions/SortDirection.java

```diff
@@ -19,14 +19,33 @@
 
 import org.apache.spark.annotation.Experimental;
 
+import static org.apache.spark.sql.connector.expressions.NullOrdering.NULLS_FIRST;
+import static org.apache.spark.sql.connector.expressions.NullOrdering.NULLS_LAST;
+
 /**
  * A sort direction used in sorting expressions.
+ * <p>
+ * Each direction has a default null ordering that is implied if no null ordering is specified
```

Comment:
This is confusing me as someone could specify the null ordering explicitly but in reality we don't leverage the default null ordering in any way.

Reply:
Never mind. The doc is correct.